### ISSUE TITLE
Practice Paper Part 3(3.02, 3.06)

### ISSUE THREAD

**Issue by xiaorui-ui** [nus-cs2103-AY2324S2/forum#1033]
I have a few questions regarding part 3 of the practice paper given.

![image](https://github.com/nus-cs2103-AY2324S2/forum/assets/87259515/bf991a55-6b5f-43c5-8347-3f2863e0af9e)

For 3.02, why is (a) not the answer? Is the accessibility necessary inside the UML diagram? Or is it that when choosing the "best match", it should have as much information as possible, provided that they are all correct? 

![image](https://github.com/nus-cs2103-AY2324S2/forum/assets/87259515/06797424-d41e-4a4d-b34e-970d94b64e66)

Similar to 3.02, is (e) superior to (c) since (e) has more information and it's correct? Or is it the correct answer due to some other reason?

Thank you!

**Comment by baskargopinath** [comment]
i think if it provides more correct info (navigability, accessibility), its a better match

**Comment by damithc** [comment]
@xiaorui-ui Yes, if you have to choose between multiple correct diagrams, choose the one that has more information.
